crux_core/src/bridge/mod.rs

```diff
@@ -29,8 +32,7 @@ where
     Eff: Effect,
     A: App,
 {
-    core: Core<Eff, A>,
-    registry: ResolveRegistry,
+    inner: BridgeInner<Eff, A, Bincode>,
```

Comment:
What's the benefit to having `BridgeInner` vs just doing this on the Bridge?

Reply:
Basically just backwards compatibility. `Inner` is a terrible name - what it should be is a `BridgeWithSerializer`.

I think we still want `Bridge` to just default to bincode like it always did, because the typegen assume is. This just opens up the option for a different serialization, if you're willing to handwrite the foreign bindings, or generate them using something else.

I'm almost tempted to hide this behind a feature flag even.